docker/cyclecloud8/scripts/cyclecloud_install.py

```diff
@@ -647,14 +651,17 @@ def main():
         install_pre_req()
         download_install_cc()
     
-    modify_cs_config(options = {'webServerMaxHeapSize': args.webServerMaxHeapSize,
+    if args.notGenerateCsConfig:
+        print("Skipping CycleCloud config generation")
+    else:
+        modify_cs_config(options = {'webServerMaxHeapSize': args.webServerMaxHeapSize,
                                 'webServerPort': args.webServerPort,
                                 'webServerSslPort': args.webServerSslPort,
                                 'webServerClusterPort': args.webServerClusterPort,
                                 'webServerEnableHttps': True,
                                 'webServerHostname': args.webServerHostname})
 
-    start_cc()
+        start_cc()
```

Comment:
is this indent intentional?  (looks like this line should NOT be indented)

Reply:
so only reason I indented is if we don't update cycle_server.properties do we need to restart CC?